core-bundle/contao/drivers/DC_Folder.php

```diff
@@ -592,6 +592,14 @@ class="tree-view"
 				data-contao--toggle-nodes-expand-all-title-value="' . $GLOBALS['TL_LANG']['DCA']['expandNodes'][1] . '"
 				data-contao--toggle-nodes-collapse-all-value="' . $GLOBALS['TL_LANG']['DCA']['collapseNodes'][0] . '"
 				data-contao--toggle-nodes-collapse-all-title-value="' . $GLOBALS['TL_LANG']['DCA']['collapseNodes'][1] . '"
+				data-contao--toggle-state-target="controls"
+				data-contao--toggle-state-active-class="filter-active"
+				data-contao--toggle-state-active-label-value="' . $GLOBALS['TL_LANG']['DCA']['toggleFilter'][2] . '"
+				data-contao--toggle-state-active-title-value="' . $GLOBALS['TL_LANG']['DCA']['toggleFilter'][3] . '"
```

Comment:
toggleFilter.3 does not exist as a translation, the mobile filter also does not work in DC_Folder for now

Reply:
Fixed in f6e638d8923da799d6d5f460079a7b155c82e3c5